app/views/monthlyContributions.scala.html

```diff
@@ -18,12 +18,12 @@
 @scripts = {
     <script type="text/javascript">
         window.guardian = window.guardian || {};
+        window.guardian.isTestUser = @isTestUser;
```

Comment:
I would not change this, because the fact of a user being test user or not is a property that concerns a user.